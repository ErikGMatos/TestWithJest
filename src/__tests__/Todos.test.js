import React from "react";
import { shallow } from "enzyme";
import Todos from "../Todos";

import configureStore from "redux-mock-store";
const mockStore = configureStore();

const initialState = {
  todos: [
    { id: 0, text: "fazer cafe" },
    { id: 1, text: "fazer codigo" },
    { id: 2, text: "fazer live" }
  ]
};

const store = mockStore(initialState);

describe("Testing Todos Component", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<Todos />, { context: { store } }).dive();

    expect(wrapper.debug()).toMatchSnapshot();
  });
});
