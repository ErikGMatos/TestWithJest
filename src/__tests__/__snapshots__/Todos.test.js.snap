// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Todos Component should render correctly 1`] = `
"<ul className=\\"todos\\">
  <li>
    fazer cafe
  </li>
  <li>
    fazer codigo
  </li>
  <li>
    fazer live
  </li>
</ul>"
`;
